android_jni/avifandroidjni/src/androidTest/java/org/aomedia/avif/android/AvifDecoderTest.java

```diff
@@ -141,7 +141,9 @@ public static List<Object[]> data() throws IOException {
   // can be decoded this way.
   @Test
   public void testDecodeUtilityClass() throws IOException {
-    if (image.isAnimated) return;
+    if (image.isAnimated) {
+      return;
+    }
```

Comment:
The current 'git cl format' command in Chrome also adds curly braces to single-line if statements.